packages/theme/src/node/hopeTheme.ts

```diff
@@ -41,7 +41,9 @@ export const hopeTheme = (
   // TODO: Change default value in v2 stable
   behavior: HopeThemeBehaviorOptions | boolean = true,
 ): ThemeFunction => {
-  checkVuePressVersion();
+  if (isPlainObject(behavior) && behavior.checkVuePress) {
```

Comment:
This is not the behavior in docs, and not the default behavior.

Current change just disable this by default

Reply:
sorry, thanks for your code